### PR TITLE
Stop call loadtest for HPC when it equal 1

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

**Comment by asmorodskyi** [comment]
@okurz  here is fix of issue which I asked you about do I need proof run for this ?

File: products/sle/main.pm [hunk 1]
```diff
@@ -1355,7 +1355,8 @@ elsif (get_var('HPC')) {
         # load hpc multimachine scenario based on value of HPC variable
         # e.g 'hpc/$testsuite_[master|slave].pm'
         my $hpc_mm_scenario = get_var('HPC');
```
Comment (asmorodskyi) [inline products/sle/main.pm]: this does not make any sense , we are inside elsif (get_var('HPC')) :)

**Comment by Soulofdestiny** [comment]
LGTM, verification run appreciated

**Comment by asmorodskyi** [comment]
http://kimball.arch.suse.de/tests/555
http://kimball.arch.suse.de/tests/556
http://kimball.arch.suse.de/tests/551
pdsh test still running

**Comment by Soulofdestiny** [comment]
pdsh tests:
http://drnek.arch.suse.de/tests/798
http://drnek.arch.suse.de/tests/799

**Pull request merged**